interface/modules/zend_modules/module/Installer/view/layout/layout.phtml

```diff
@@ -35,8 +35,7 @@ echo $this->doctype(); ?>
         <!-- Scripts -->
         <?php echo $this->headScript()
 					->prependFile($this->basePath() . '/js/application/common.js')
-          ->prependFile($this->basePath() . '/js/lib/html5.js', 'text/javascript', array('conditional' => 'lt IE 9',))
-					->prependFile($this->basePath() . '/js/lib/bootstrap.min.js')
+                    ->prependFile($this->basePath() . '/js/lib/bootstrap.min.js')
```

Comment:
Testing: We should at least point to /public/assets in this file so we can get rid of the excess files.